test/test-fs.c

```diff
@@ -3943,9 +3959,7 @@ TEST_IMPL(open_osfhandle_valid_handle) {
   loop = uv_default_loop();
 
   r = uv_fs_open(NULL,
-                 &open_req1,
-                 "test_file",
-                 O_RDWR | O_CREAT,
+                 &open_req1, "test_file", UV_FS_O_RDWR | UV_FS_O_CREAT,
```

Comment:
Style not: ideally we could keep libuv-style with one-arg per line

Reply:
Though that style is not consistent, see for example: https://github.com/libuv/libuv/blob/4dc9fd7d7e9cb3aa2b4772bb24bbe0dba4df6d75/test/test-fs-copyfile.c#L78

Also, what is up with libuv not having a `.clang-format` in this day and age? :)